README.md

```diff
@@ -176,7 +178,7 @@ jwt.sign({ userId: user.id })
 
 ## Security
 
-This microservice is intended to be very secure.
+This microservice is intented to be very secure.  User accounts can be limited to certain domains by configuring the `WHITELISTED_DOMAINS` env variable.
```

Comment:
yeah I saw a conflict on that line but for some reason my eyes couldn't detect that change - thanks for pointing that out : )